modules/stanford_profile_styles/lib/scss/component/paragraph/wysiwyg/_headings.scss

```diff
@@ -75,5 +75,37 @@
       }
     }
   }
+
+  // Overide the heading link icons for the new Localist widget replacing the old
+  // Event calendar. This is to keep the whole card clickable.
+  .localist-widget {
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
```

Comment:
What if we had a `headings` mixin? Maybe something to think about like this:
```
@mixin headings {
    h2, h3,
    h4, h5, h6 {
        @content;
    }
}
```

Reply:
Oh that's a good idea. I will play around with it.